js/hubs/components/AudioPlayer.js

```diff
@@ -119,12 +119,18 @@ const HubAudioPlayer = ({ hubPubkey }) => {
                     <Typography>
                       Now Playing:{' '}
                       <Link
-                        href={`/${track.hubHandle}/${
-                          track.hubPostPubkey ? 'posts' : 'releases'
+                        href={`/${hubState[track.hub].handle}/${
+                          releaseState.metadata[track.releasePubkey]
+                            .hubPostPubkey
+                            ? 'posts'
+                            : 'releases'
                         }/${
-                          track.hubPostPubkey
-                            ? track.hubPostPubkey
-                            : track.hubReleaseId
+                          releaseState.metadata[track.releasePubkey]
```

Comment:
`releaseState.metadata[track.releasePubkey]` should be refactored to 1 var as it is repeated 4 times